save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/InputForms.kt

```diff
@@ -17,7 +17,7 @@ import kotlinx.html.js.onChangeFunction
  * @property str
  */
 @Suppress("WRONG_DECLARATIONS_ORDER")
-internal enum class InputTypes(val str: String) {
+enum class InputTypes(val str: String) {
```

Comment:
Why did you remove `internal`?

Reply:
I can't use internal class in external props interface

Reply:
Oh, OK